Stats within WP dashboard #17

mehul0810 opened the pull request:
This PR resolves #17 

I have added a new admin page under `Dashboard` named `Analytics` to show the Embed Analytics page of Plausible. 

**Analytics/Statistics Page - when settings configured properly**
![image](https://user-images.githubusercontent.com/1852711/114281761-dd473700-9a5d-11eb-9eaa-be347e182562.png)

**Analytics/Statistics Page - when settings  NOT configured properly**
![image](https://user-images.githubusercontent.com/1852711/114282326-cce48b80-9a60-11eb-8172-39897308a37f.png)

**Embed Analytics Settings**
![image](https://user-images.githubusercontent.com/1852711/114281787-08318b00-9a5e-11eb-9c23-cef7fb60d048.png)

Let me know if there are any text changes.

metmarkosaric commented:
Oh nice, thanks @mehul0810!

Perhaps we can do the steps like in the instructions for the custom domain? Perhaps we can add a direct link to Dashboard -> Analytics here too.

Heading: **View your stats in your WordPress dashboard**

1. Create a secure and private shared link in your Plausible account. [See how »](https://plausible.io/docs/shared-links)
2. Enable this setting and paste your shared link to view your stats in your WordPress dashboard.

mehul0810 commented:
@metmarkosaric I have implemented the changes as per your suggestions. Also, added 3rd step of instruction to link to the analytics page. See screenshot below:

![image](https://user-images.githubusercontent.com/1852711/115070821-1296cd80-9f13-11eb-8871-cb5992e939a4.png)

metmarkosaric commented:
looks good to me @mehul0810, thank you!
